### PR TITLE
Feature : Added custom scrollbar

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
## Description 📝

Closes : #270 

## Changes 💬

- Added custom scrollbar with track, thumb and root color property
- Added border-radius and color hover effect on hover and smooth transition
- Looks visually appealing in both dark and light themes

## Checklist ✅

- [x] Reviewed the code changes thoroughly.
- [x] Tested the changes locally and ensured they work as expected.
- [x] Updated relevant documentation.
- [x] Added appropriate test cases, if applicable.
- [x] Considered potential security or performance implications.
- [x] Code adheres to the project's style guidelines.
- [x] Addressed all reviewer comments and concerns, if any.

## Screenshots (if applicable)

https://github.com/PiyushKalyanpy/GyanaGuru/assets/115401171/e3ed662d-7151-4bc3-a433-fab848c64ed9

#### I'm a gssoc contributor so please add required gssoc labels
image

### CODE REVIEW

**Review by lalitmee — APPROVED** [review]
LGTM

File: styles/globals.css [hunk 1]
```diff
@@ -44,8 +45,26 @@ p, a, li, span, input, textarea, button {
...
+}
+
+::-webkit-scrollbar-thumb {
+  background: #908f8f;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #c8c7c7;
+  border-radius: 8px;
 }
```
[the remark this answers is not in the document]
Comment (ujjawaltyagii) [reply styles/globals.css]: Done !! 👍

**Comment by ujjawaltyagii** [comment]
@PiyushKalyanpy Done with the changes you suggested ! NOW <b>color of scrollbar-thumb and scrollbar-track </b> both changes with change in theme.


https://github.com/PiyushKalyanpy/GyanaGuru/assets/115401171/d1c1e38a-db6f-447d-af0f-4c65992d994c

**Review by PiyushKalyanpy — APPROVED** [review]

**Comment by ujjawaltyagii** [comment]
@lalitmee please add gssoc labels and levels so that it can count for my gssoc entry (also add assignee in this PR to me)

**Pull request merged**